.idea/deploymentTargetDropDown.xml

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Changing this file previously caused the CI to fail for me but I am not sure what deleting it will do.

Reply:
I did not intend to delete it. This is a problem in my gitignore.